SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialStaticComponentView.cpp

```diff
@@ -14,6 +14,7 @@
 #include "Schema/ServerRPCEndpointLegacy.h"
 #include "Schema/Singleton.h"
 #include "Schema/SpawnData.h"
+#include "Schema/SpatialDebugging.h"
```

Comment:
NIT this should be a line above to be alphabetized

Reply:
The alphabet is hard.